examples/allocation-endpoint/README.md

```diff
@@ -0,0 +1,104 @@
+# Allocation Endpoint
+
+This is a sample solution to enable an Allocation Endpoint proxy in front of GKE Agones clusters.
+
+In this folder you will find:
+
+1. Terraform modules that creates resources in GCP
+2. The Allocation Endpoint Proxy code that gets deployed to Cloud Run
+3. An ESP container that gets deployed to agones-allocator
+4. A client sample code to send allocation requests to the proxy
+5. Dockerfile that builds the proxy and scripts to push the image to a docker repository
+6. Documentation on how to use the solution.
+
+Here is the architecture of GCP resources created:
+
+![architecture](https://github.com/googleforgames/agones/blob/main/examples/allocation-endpoint/architecture.png?raw=true)
+
+
+## GKE cluster
+First and foremost you need to create clusters and installing Agones for experimenting with this solution.
+The clusters can be in the same GCP project as your Allocation Endpoint proxy or it can be in different project.
+
+---
+**NOTE**
+
+The solution is not experimented with non-GKE clusters.
+
+---
+
+When creating GKE Clusters, [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) needs to be enabled.
+
+<pre>
+gcloud container clusters create [NAME] --cluster-version=1.21 \
```

Comment:
This command is going to get out of date relative to https://agones.dev/site/docs/installation/creating-cluster/gke/ after the next k8s upgrade. Would it be possible to link to that page and just show the additional flag that needs to be passed in?